portfolio/src/main/webapp/src/com.js

```diff
@@ -90,16 +130,47 @@ function addCommentsToPage(comments) {
  *
  */
 function updateComments() {
-  getCommentsFromServer()
+  // Extract max comments per page from drop-down
+  const elt = document.getElementById('comment-number');
+  const maxComments = elt.options[elt.selectedIndex].value;
+
+  getCommentsFromServer(maxComments)
     .then(addCommentsToPage)
     .catch((error) => {
       const errorCard = createCommentCard(
         '<b>Unable to fetch comments from server</b>',
       );
+      document.getElementById('comment-list').innerHTML = '';
       document.getElementById('comment-list').appendChild(errorCard);
       console.error(error);
     });
 }
 
+/**
+ * Deletes all comments from the server and reloads the page
+ *
+ */
+function deleteAllComments() {
```

Comment:
I'd recommend renaming this to something like `handleDeleteComments` which would make it obvious its an event handler.